### PR TITLE
Added GetPlaylistUri to APP and /play to FE

### PR DESCRIPTION
**Pull request by peddybeats**

### CODE REVIEW

File: fe/src/views/play.jade [hunk 1]
```diff
@@ -0,0 +1 @@
+iframe(src="https://embed.spotify.com/?uri=" + uri, width="100%", height="100%", frameborder="0", allowtransparency="true")
```
Comment (peddybeats) [inline fe/src/views/play.jade:1]: Make responsive

File: fe/src/http_handler.js [hunk 2]
```diff
@@ -172,18 +176,41 @@ exports.generate = function(req, res) {
   if (audioFeatures == {})
     return echoBack("No audio feature parameters were supplied", null, res);
 
-  // Make RPC to APP
+  // Make GenerateJukebox RPC call
```
Comment (peddybeats) [inline fe/src/http_handler.js:179]: Do error catching here: if more than 5 seed tracks, just take the first 5.

File: fe/src/views/play.jade (hunk 1 above)
Comment (peddybeats) [reply fe/src/views/play.jade:1]: won't fix now

File: fe/src/http_handler.js (hunk 2 above)
Comment (PedramPejman) [inline fe/src/http_handler.js]: Won't do now.

**Review by peddybeats — COMMENTED** [review]
LGTM

**Pull request merged**